### PR TITLE
Replace AttributeContainer with MetadataContainer with dedicated sub-module

### PR DESCRIPTION
**Pull request by shnela**

### CODE REVIEW

File: neptune/new/__init__.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
     NeptuneUninitializedException,
     NeptunePossibleLegacyUsageException,
 )
-from neptune.new.run import Run
+from neptune.new.attributes_containers import Run
```
Comment (shnela) [inline neptune/new/__init__.py]: Maybe we should call this module `containers` so the imports are shorter?

File: neptune/new/attributes_containers/__init__.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from neptune.new.attributes_containers.attribute_container import AttributeContainer
+from neptune.new.attributes_containers.handler import Handler
```
Comment (shnela) [inline neptune/new/attributes_containers/__init__.py:18]: `Handler` is something in the middle between `Attribute` and `AttributeContainer`, I'm not 100% sure about moving it here.

File: neptune/new/attributes_containers/__init__.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+from neptune.new.attributes_containers.model import Model
+from neptune.new.attributes_containers.model_version import ModelVersion
+
+# from neptune.new.attributes_containers.model_versions_table import (
+#     ModelVersionsTable,
+#     ModelVersionsTableEntry,
+# )
+from neptune.new.attributes_containers.project import Project
+from neptune.new.attributes_containers.run import Run
```
Comment (shnela) [inline neptune/new/attributes_containers/__init__.py:27]: I think that moving generic `AttributeContainer`, `Model`, `Run` etc as `module-level` import is good idea, to simplify imports in other parts of code.

I'm not 100% sure about placing `*Table` and `*TableEntry` here.

File: neptune/new/__init__.py [hunk 4]
```diff
@@ -32,7 +32,7 @@
     NeptuneUninitializedException,
     NeptunePossibleLegacyUsageException,
 )
-from neptune.new.run import Run
+from neptune.new.attributes_containers import Run, Project, Model, ModelVersion
```
Comment (shnela) [inline neptune/new/__init__.py:35]: Should we make `*Table` and `*TableEntry` objects importable for our users as well?

File: neptune/new/attributes/atoms/string.py [hunk 5]
```diff
@@ -24,7 +24,7 @@
 from neptune.new.types.atoms.string import String as StringVal
 
 if typing.TYPE_CHECKING:
-    from neptune.new.attribute_container import AttributeContainer
+    from neptune.new.attributes_containers import AttributeContainer
```
Comment (pkasprzyk) [inline neptune/new/attributes/atoms/string.py:27]: I have to admit: plural "attributes containers" does sound BAD for me (vs attribute container)

File: neptune/new/__init__.py [hunk 6]
```diff
@@ -32,7 +32,7 @@
     NeptuneUninitializedException,
     NeptunePossibleLegacyUsageException,
 )
-from neptune.new.run import Run
+from neptune.new.metadata_containers import Run, Project, Model, ModelVersion
```
Comment (pkasprzyk) [inline neptune/new/__init__.py:35]: Haven't we come to a conclusion NOT to do this?

File: neptune/new/runs_table.py [hunk 7]
```diff
@@ -17,44 +17,20 @@
...
-            attributes=attributes,
-        )
-
-
-class RunsTable(AttributesContainersTable):
-    table_entry_cls = RunsTableEntry
-
-    def to_runs(self):
-        return self.to_table_entries()
+from neptune.new.metadata_containers.runs_table import RunsTable, RunsTableEntry
```
Comment (pkasprzyk) [inline neptune/new/runs_table.py:36]: I know you have a separate issue for deprecation https://neptune-labs.atlassian.net/browse/NPT-11395 , but please either leave an easily greppable comment in files like that or add them to NPT-11395 description

File: neptune/new/__init__.py (hunk 6 above)
Comment (shnela) [reply neptune/new/__init__.py:35]: Yes we did, I'l fix.

File: neptune/new/runs_table.py (hunk 7 above)
Comment (shnela) [reply neptune/new/runs_table.py:36]: Isn't

```
# backwards compatibility
# pylint: disable=unused-import,wrong-import-order
```

enough to find such places?

Comment (pkasprzyk) [reply neptune/new/runs_table.py:36]: right, sorry -- I've seen you've removed from https://github.com/neptune-ai/neptune-client/pull/811/files#diff-51310aa42e9441fc7625038b9dffa9f353864634268799198b624f273f74f4ecL29 and got lost a little bit

**Review by pkasprzyk — APPROVED** [review]

**Pull request merged**